mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -244,12 +254,28 @@ public void setAttribute(String name, Object value) {
     }
 
     public void log(LogLevel level, String message) {
-        System.out.println("[" + level + "]" + message);
+        if (logger != null) {
+            switch (level) {
+            case INFO:
+                logger.info(message);
+                break;
+            case WARN:
+                logger.warn(message);
+                break;
+            case ERROR:
+                logger.error(message);
+                break;
+            default:
+                logger.debug(message);
+            }
+        } else {
+            System.out.println("[" + level + "]" + message);
+        }
     }
 
     public void log(LogLevel level, String message, Throwable t) {
-        System.out.println("[" + level + "]" + message);
-        t.printStackTrace(System.out);
+        log(level, message);
+        log(level, t.getMessage());
```

Comment:
Yes, but it's a test implementation; and i don't want to verfiy each stack in tests.